decidim-core/lib/decidim/seeds.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "faker"
+
+module Decidim
+  # Base class to be inherited from the different modules' seeds classes
+  class Seeds
+    protected
+
+    def seeds_root = File.join(__dir__, "..", "..", "db", "seeds")
+
+    def hero_image = create_image!(seeds_file: "city.jpeg", filename: "hero_image.jpeg")
+
+    def banner_image = create_image!(seeds_file: "city2.jpeg", filename: "banner_image.jpeg")
+
+    def create_attachment(attached_to:, filename:, attachment_collection: nil)
+      content_type = {
+        jpg: "image/jpeg",
+        jpeg: "image/jpeg",
+        pdf: "application/pdf"
+      }[filename.split(".")[1].to_sym]
+
+      Decidim::Attachment.create!(
+        title: Decidim::Faker::Localized.sentence(word_count: 2),
+        description: Decidim::Faker::Localized.sentence(word_count: 5),
+        attachment_collection:,
+        attached_to:,
+        content_type:,
+        file: create_image!(seeds_file: filename, filename:) # Keep after attached_to
```

Comment:
I would pass the content_type also to the `create_blob` method, to avoid creating PDF blobs with "image/jpeg" content type.

```suggestion
        file: create_blob!(seeds_file: filename, filename:, content_type: ) # Keep after attached_to
```